src/ex1/script.js

```diff
@@ -0,0 +1,143 @@
+const userInput = document.querySelector(".userinput");
+const userList = document.querySelector(".userlist");
+const taskBtn = document.querySelector(".todobutton");
+const item = document.querySelector(".userlist");
+const deleteAllBtn = document.querySelector(".footer button");
+
+document.addEventListener("DOMContentLoaded", getTasks);
+taskBtn.addEventListener("click", addTask);
+userList.addEventListener("click", deleteTask);
+deleteAllBtn.addEventListener("click", deleteAll);
+item.addEventListener("click", taskClick);
+
+
+function deleteTask(e) {
+  const item = e.target;
+  let todos;
+  if (localStorage.getItem("todos") === null) {
```

Comment:
youre using this block of code a lot. why not extract it to a method?

if (localStorage.getItem("todos") === null) {
    todos = [];
  } else {
    todos = JSON.parse(localStorage.getItem("todos"));
  }